src/custom-element-utilities.ts

```diff
@@ -50,9 +50,16 @@ export const createWebComponentClassFromBehavior = (
     _bindings: Binding[];
     /**@internal */
     au: { controller: Controller; };
+    /**@internal */
+    initialised = false;
```

Comment:
let's move initialization to constructor body, it's easier when you don't have to read the whole class body to know what's available

Reply:
Done!